js_modules/dagster-ui/packages/ui-core/src/assets/AssetsOverview.tsx

```diff
@@ -0,0 +1,335 @@
+        borderTop: `1px solid ${Colors.backgroundGray()}`,
+      }}
+    >
+      <SectionName>{sectionName}</SectionName>
+    </Box>
+  );
+};
+
+const SectionBody = ({children}: {children: React.ReactNode}) => {
+  return (
+    <Box
+      padding={{horizontal: 24, vertical: 16}}
+      flex={{wrap: 'wrap'}}
+      style={{rowGap: '14px', columnGap: '24px'}}
+    >
+      {children}
+    </Box>
+  );
+};
+
+const LinkToAssetGraphGroup = (groupMetadata: GroupMetadata) => {
+  return `/asset-groups?groups=[${encodeURIComponent(JSON.stringify(groupMetadata))}]`;
+};
+
+const LinkToAssetGraphComputeKind = (computeKind: string) => {
+  return `/asset-groups?computeKindTags=${encodeURIComponent(JSON.stringify([computeKind]))}`;
+};
+
+const LinkToCodeLocation = (repoAddress: RepoAddress) => {
+  return `/locations/${repoAddressAsURLString(repoAddress)}/assets`;
+};
+
+export const AssetsOverview = ({viewerName}: {viewerName?: string}) => {
+  useTrackPageView();
+
+  const params = useParams();
+  const currentPath: string[] = ((params as any)['0'] || '')
+    .split('/')
+    .filter((x: string) => x)
+    .map(decodeURIComponent);
+
+  useDocumentTitle('Assets');
+
+  const assetsQuery = useQuery<AssetCatalogTableQuery, AssetCatalogTableQueryVariables>(
+    ASSET_CATALOG_TABLE_QUERY,
+    {
+      notifyOnNetworkStatusChange: true,
+    },
+  );
+  const assetCountBySection = buildAssetCountBySection(
+    assetsQuery.data?.assetsOrError.__typename === 'AssetConnection'
+      ? assetsQuery.data.assetsOrError.nodes
+      : [],
+  );
+  const {UserDisplay} = useLaunchPadHooks();
+
+  if (assetsQuery.loading) {
+    return (
+      <Page>
+        <AssetPageHeader assetKey={{path: currentPath}} />
+        <Box flex={{direction: 'row', justifyContent: 'center'}} style={{paddingTop: '100px'}}>
+          <Box flex={{direction: 'row', alignItems: 'center', gap: 16}}>
+            <Spinner purpose="body-text" />
+            <div style={{color: Colors.textLight()}}>Loading assets…</div>
+          </Box>
+        </Box>
+      </Page>
+    );
+  }
+
+  return (
+    <Box flex={{direction: 'column'}} style={{height: '100%', overflow: 'hidden'}}>
+      <AssetPageHeader
+        assetKey={{path: currentPath}}
+        right={
+          <Box flex={{gap: 12, alignItems: 'center'}}>
+            <ReloadAllButton label="Reload definitions" />
+          </Box>
+        }
+      />
+      <Box
+        padding={64}
+        flex={{justifyContent: 'center', alignItems: 'center'}}
+        style={{overflowY: 'scroll'}}
+      >
+        <Box style={{width: '60%'}} flex={{direction: 'column', gap: 16}}>
+          <Box flex={{direction: 'row', alignItems: 'center', justifyContent: 'space-between'}}>
+            <Heading>
+              {getGreeting()}
+              {viewerName ? `, ${viewerName}` : ''}
+            </Heading>
+            <Box flex={{direction: 'row', gap: 12, alignItems: 'center'}}>
+              <ViewAllAssetsLink />
```

Comment:
It looks like there's already a `gap: 12` between the link and the button, so perhaps the `gap: 4` on the link isn't needed? Or maybe this should be `gap: 16`?

Reply:
Ah good call, I think `gap: 16` is right